TheWarWithin/DruidFeral.lua

```diff
@@ -2700,7 +2700,7 @@ spec:RegisterAbilities( {
             gain( talent.berserk.enabled and 2 or 1 + ( allow_crit_prediction and crit_pct_current * active_enemies >= 230 and 1 or 0 ), "combo_points" )
 
             if talent.bloodtalons.enabled then
-                applyBuff( "bt_swipe_cat" )
+                applyBuff( "bt_swipe" )
```

Comment:
Can you also add `copy = "bt_swipe_cat",` to the `bt_swipe` fake aura definition?

Reply:
Added the thrash equivalent as well for completeness